services/distributions_api/routes/deployments.py

```diff
@@ -143,15 +147,14 @@ async def create_deployment(
             db.rollback()
             raise HTTPException(status_code=status.HTTP_425_TOO_EARLY, detail="Deployment is already in progress!")
 
+    task_id = None
     if not payload.error:
-        background_tasks.add_task(
-            run_deployer,
+        task_id = tasks.run_deployer_task.delay(
             dist=dream_dist,
-            # port=port,
             deployment_id=deployment.id,
-        )
+        ).id
 
-    return schemas.DeploymentRead.from_orm(deployment)
+    return {"task_id": task_id, **schemas.DeploymentRead.from_orm(deployment).__dict__}
```

Comment:
front gets task_id and can get the status of the task (states are quite poor: "pending" stands for `I don't know the task`, "started", "failure" & "success"